### ISSUE TITLE
Remove hashing for unicode identifiers once skulpt supports it

### ISSUE THREAD

**Issue by bjorn3** [Felienne/hedy#959]
Once https://github.com/skulpt/skulpt/pull/1386 lands skulpt will have native support for unicode identifiers.

**Comment by eddieantonio** [comment]
Is this issue related to the follow issue I encountered in the Spanish localization?

<img width="507" alt="Hedy code with an error" src="https://user-images.githubusercontent.com/2294397/146686206-9d7bf9f4-fa54-4742-8b5a-d561fa50d3d6.png">

> Algo salió mal mientras se ejecutaba el programa.
>
> (_Something went wrong while running the program_)

I have declared a variable in Spanish "ángulo", and Hedy returns some compiler-speaker — `bad token: T_OP` which is incomprehensible to someone who's written compilers :) and I'm afraid impeterably confusing for Spanish speakers.

If I'm using Hedy in Spanish, I should be able to use Spanish identifiers!

**Comment by bjorn3** [comment]
This is somewhat related, though not exactly this issue. In this case `girar ángul` should result in `ángul` being hashed, but the generated python contains `t.right(ángulo)` rather than `t.right(v7fa75eeaae3db0dba100b4e8f72615c8)`, while `ángulo is 90` does get translated to `v7fa75eeaae3db0dba100b4e8f72615c8 = '90'`.

**Comment by Felienne** [comment]
> Is this issue related to the follow issue I encountered in the Spanish localization?
> 
> <img alt="Hedy code with an error" width="507" src="https://user-images.githubusercontent.com/2294397/146686206-9d7bf9f4-fa54-4742-8b5a-d561fa50d3d6.png">
> 
> > Algo salió mal mientras se ejecutaba el programa.
> > (_Something went wrong while running the program_)
> 
> I have declared a variable in Spanish "ángulo", and Hedy returns some compiler-speaker — `bad token: T_OP` which is incomprehensible to someone who's written compilers :) and I'm afraid impeterably confusing for Spanish speakers.
> 
> If I'm using Hedy in Spanish, I should be able to use Spanish identifiers!

Firstly, sorry about the error message @eddieantonio, that is incomprehensible indeed! And yes, Spanish identifiers should absolutely work! It seems indeed that we forget to hash a var somewhere (sadly that is needed because of the Skulpt issue outlined above) 

I will dive into this tomorrow.

**Comment by Felienne** [comment]
(Sad to see no progress on that on Skulpt's side :( )

**Comment by eddieantonio** [comment]
> This is somewhat related, though not exactly this issue. In this case `girar ángul` should result in `ángul` being hashed, but the generated python contains `t.right(ángulo)` rather than `t.right(v7fa75eeaae3db0dba100b4e8f72615c8)`, while `ángulo is 90` does get translated to `v7fa75eeaae3db0dba100b4e8f72615c8 = '90'`.

Is this linked to an existing issue, or a brand new one? 😂

Oh dear, removing hashing will also make for more transparent error messages! I can assume some error can be emitted that result in "unknown identifier v7fa75eeaae3db0dba100b4e8f72615c8" or something similar 😭 so hopefully skulpt makes progress on this!

**Comment by Felienne** [comment]
> > This is somewhat related, though not exactly this issue. In this case `girar ángul` should result in `ángul` being hashed, but the generated python contains `t.right(ángulo)` rather than `t.right(v7fa75eeaae3db0dba100b4e8f72615c8)`, while `ángulo is 90` does get translated to `v7fa75eeaae3db0dba100b4e8f72615c8 = '90'`.
> 
> Is this linked to an existing issue, or a brand new one? 😂

I think it is new. Based on @bjorn3's observation it seems we forget to hash somewhere (in the `turn` command, most likely). It is unrelated to Spanish keywords:

![image](https://user-images.githubusercontent.com/1003685/146687303-326b1cf3-b68a-42f0-910d-e548f64d64f4.png)

> Oh dear, removing hashing will also make for more transparent error messages! I can assume some error can be emitted that result in "unknown identifier v7fa75eeaae3db0dba100b4e8f72615c8" or something similar 😭 so hopefully skulpt makes progress on this!

Yes we are looking forward to that code being no longer needed a lot!

**Comment by Felienne** [comment]
I think your issue should be fixed by this PR: https://github.com/Felienne/hedy/pull/1563

Thanks @bjorn3, your diagnosis helped me realize what the issue was and also that I could fix it in 5 mins on a Sunday night. I set it to automerge so if you all agree it works it can be merged straight away.

**Comment by eddieantonio** [comment]
Ah, thanks @Felienne! That was quick!
